website/common/script/ops/changeClass.js

```diff
@@ -44,26 +43,6 @@ module.exports = function changeClass (user, req = {}, analytics) {
     user.flags.classSelected = true;
 
     each(['weapon', 'armor', 'shield', 'head'], (type) => {
```

Comment:
agreed the code now only runs for `(type === 'weapon' || (type === 'shield' && klass === 'rogue')`